R/type_density.R

```diff
@@ -1,237 +1,132 @@
-#' @importFrom methods as
-#' @importFrom stats update density
-#' @rdname tinyplot
+#' Density plot type
+#' 
+#' @md
+#' @description Type function for density plots.
+#' @inheritParams stats::density
+#' @param kernel a character string giving the smoothing kernel to be used. This
+#'   must partially match one of `"gaussian"`, `"rectangular"`, `"triangular"`,
+#'   `"epanechnikov"`, `"biweight"`, `"cosine"` or `"optcosine"`, with default
+#'   `"gaussian"`, and may be abbreviated to a unique prefix (single letter).
+#'
+#'   `"cosine"` is smoother than `"optcosine"`, which is the usual 'cosine'
+#'   kernel in the literature and almost MSE-efficient. However, `"cosine"` is
+#'   the version used by S.
+#' @inheritParams type_ribbon
+#' @param indiv.bw Should the smoothing bandwidth be computed individually
```

Comment:
Is this a standard arg name? If not, I might reverse the order in case there are other `bw`-related arguments in the future. That would allow us to clump them together in alphabetical-looking order. Also, if I'm looking for bandwidth options, I'll probably look for "b" rather than "i"